ably/options.go

```diff
@@ -393,3 +386,41 @@ func (p *PaginateParams) EncodeValues(out *url.Values) error {
 	p.ScopeParams.EncodeValues(out)
 	return nil
 }
+
+type ClientOptionsV12 []func(*ClientOptions)
+
+// TokenLikeV12 is something that can be interpreted as a TokenDetails.
+type TokenLikeV12 interface {
```

Comment:
Both Token and TokenLike are somewhat bad because a TokenRequest isn't a token, it's something that turns into a token. The others don't need to be turned into a token because they already are.

So, maybe... Tokener? Or just AuthCallbackResult?

Reply:
`TokenResponse`

or
`TokenResult`
`TokenResource`

Reply:
I went with `Tokener` at 52554d2.